stable/airflow/README.md

```diff
@@ -236,11 +237,12 @@ workers:
   celery:
     instances: 10
 
-    # wait until all tasks are finished before SIGTERM of Pod
+    ## wait at most 10min for running tasks to complete
     gracefullTermination: true
+    gracefullTerminationPeriod: 600
```

Comment:
Can you document it as well in the table bellow please ?

Reply:
Since a version a few months ago, we don't list every single value in that table (only the top level ones), this one is covered by `workers.celery.*` for example.

Reply:
@gsemet is this ok?

Reply:
Ok